draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -2028,1226 +1972,1147 @@ uCDN-A                         dCDN-B                         dCDN-C
+{
+  "generic-trigger-extension-type": "execution-policy",
+  "generic-trigger-extension-value":
+  {
+    "priority": 100,
+    "depends":  [ 
+      "https://dcdn.example.com/triggers/100",
+      "https://dcdn.example.com/triggers/101"
+    ], 
+    "urgent": true
+  }
+}
+       ]]></sourcecode>
    </t>
 
-   <t>
-   In the case of uCDN accessing ObjectList objects referencing external URLs published by dCDN, both parties should comply with the CI/T interface security requirements (see <xref target="authentication-authorization"/> for details). 
-   When dCDN accesses external URLs referenced by ObjectLists supplied by uCDN, for example as part of Trigger Spec, dCDN MUST match these URLs with source metadata objects, published by uCDN, such as SourceMetadata objects specified in Section 4.2.1 of <xref target="RFC8006"/>, and use these metadata objects for content acquisition if a match was found.
-   </t>
 
+   </section>
+   </section>
+   </section>
+
+   <section title="Trigger State" anchor="trigger-state">
    <t>
-ObjectLists MAY combine regular objects and secondary ObjectLists in the same object.
-      Please note that when embedding non-JSON object lists directly in ObjectList, absolute URLs MUST be provided at all times and the text SHOULD be encoded in accordance with the JSON grammar specification <xref target="ECMA404"/>, including explicit newline encoding. When uCDN accesses ObjectList metadata resources published by uCDN, the same interface authentication and authorization requirements would apply, as when accessing the interface itself.
+	Trigger state describes the current state of the triggered activity. 
+	It MUST be one of the JSON strings in the following table:
    </t>
 
-   <t>ObjectList is encoded as an array of per-object records in JSON format as follows:</t>
-   <t>External HLS manifest:</t>
-   <sourcecode type="json" name="external-hls-manifest"><![CDATA[
-[
-  { 
-    "href": "https://example.com/hls/a36f764e/index.m3u8",
-    "type": "hls" 
-  }
-]
-]]></sourcecode>
+   <texttable style="full"><ttcol align="left">JSON String</ttcol>
+   <ttcol align="left"> Description </ttcol>
+   <c>pending</c>
+   <c>The trigger has not yet been acted upon.</c>
+   <c>active</c>
+   <c>The trigger is currently being acted upon.</c>
+   <c>complete</c>
+   <c>The trigger processing completed successfully.</c>
+   <c>processed</c>
+   <c>The trigger has been created, and no further status update 
+	   will be made (can be used in cases where completion cannot be confirmed).</c>
+   <c>failed</c>
+   <c>The trigger processing could not be completed.</c>
+   <c>cancelling</c>
+   <c>The trigger processing is still in progress, but the trigger has been cancelled by uCDN.</c>
+   <c>cancelled</c>
+   <c>The trigger was cancelled by uCDN.</c>
+   </texttable>
+   
+   <t>
+   	Along with the trigger state, the Trigger resource has a state reason property,
```

Comment:
Nit: Maybe change "state reason" to "state-reason" (maybe in quotes), so that it references the actual property name.

Reply:
Note that this is still open and recommended.  I didn't see a correction for it.